src/config.ts

```diff
@@ -1,9 +1,9 @@
 import { ProcessorConfig } from './types'
 import { lookupArchive } from '@subsquid/archive-registry'
 import axios from 'axios'
-export type Network = 'foucoco' | 'amplitude' | 'pendulum'
-export const network: Network =
-    <'foucoco' | 'amplitude' | 'pendulum'>process.env.NETWORK || 'amplitude'
+export type Network = 'foucoco' | 'amplitude' | 'pendulum' | 'local'
+export const network = (process.env.NETWORK as Network) || 'amplitude'
```

Comment:
Alternatively, we could define `network` to be `foucoco` if `process.env.NETWORK` is `local` and only check for equality if `process.env.NETWORK` with `local` when selecting a `ProcessorConfig` [below](https://github.com/pendulum-chain/pendulum-squids/pull/52/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R51).

That way we don't need to change any of the other code that checks the value of the variable `network`.

Reply:
I changed the logic: https://github.com/pendulum-chain/pendulum-squids/pull/52/commits/84d220c8ce2d0fa32da16822f50446ba8db77843